integrations/standalone/src/mock.tsx

```diff
@@ -5,12 +5,12 @@ import * as monaco from 'monaco-editor/esm/vs/editor/editor.api';
 import React from 'react';
 import ReactDOM from 'react-dom/client';
 import { URLParams } from './url-helper';
-import { InscriptionEditorType } from '@axonivy/inscription-protocol';
+import { InscriptionType } from '@axonivy/inscription-protocol';
 
 export async function start(): Promise<void> {
   const theme = URLParams.themeMode();
   const readonly = URLParams.parameter('readonly') ? true : false;
-  const type = (URLParams.parameter('type') as InscriptionEditorType) ?? undefined;
+  const type = (URLParams.parameter('type') as InscriptionType["id"]) ?? undefined;
```

Comment:
don't know what this "id" is, but we normally use single quotes in js. Also something which prettier should fix I think?

Reply:
this is the enumeration that lists all available 'element-types'. VS-Code helped me to write that 'id' quote, gonna try again with prettier